gen_template.rb

```diff
@@ -82,6 +82,29 @@ def gen_template(
           ['-m', '{{ user `memory` }}'],
         ],
       ),
+      builder(
+        type: 'hyperv-iso',
+        generation: 1,
+        # generation: 2, # gen 2 fails uefi boot, missing configuration in builders/hyperv-iso.nix
```

Comment:
Packer builds just fine with `generation 2` set but the resultant vagrant `.box` fails at UEFI boot.

Guessing something I missed re: the bootloader and UEFI in `scripts/builders/hyperv-iso.nix`... was hoping a maintainer or someone from the community may have a better idea here. Gen v1 works no problem however.